content/docs/sidebar.json

```
@@ -87,6 +87,18 @@
         "slug": "what-is-dvc",
         "source": "what-is-dvc.md"
       },
+      {
+        "slug": "concepts",
+        "source": false,
+        "children": [
+          "cache",
+          "dvc-files",
+          "metrics-plots",
+          "pipelines",
+          "remote",
+          "workspace"
```

Ordered from more to less basic:
```suggestion
          "workspace",
          "dvc-metafiles",
          "cache",
          "pipelines",
          "metrics-plots",
          "remote"
```

---

Thanks for the notes on order... I'll update these to follow the labels in the issue description.

---

K. And actually remote is probably more basic than metrics/plots.